doc/BUILD.md

```diff
@@ -57,13 +57,20 @@ Fetch, build and install turtlebot3 packages:
 ```console
 $ mkdir -p ~/turtlebot3_ws/src
 $ cd ~/turtlebot3_ws
-$ wget https://raw.githubusercontent.com/ROBOTIS-GIT/turtlebot3/ros2/turtlebot3.repos
+$ wget https://raw.githubusercontent.com/ROBOTIS-GIT/turtlebot3/eloquent-devel/turtlebot3.repos
 $ vcs import src < turtlebot3.repos
 $ rosdep install -y -r -q --from-paths src --ignore-src --rosdistro <ros2-distro-name>
 e.g- ros2-distro-name = dashing
 $ colcon build --symlink-install
 ```
 
+### 2.4 Quick test
```

Comment:
Sorry for that, but I totally forgot to add the "   ``` console    " before the commands line bellow.

Reply:
Just submit a new PR with that